etc/skeleton/.travis.yml

```diff
@@ -1,20 +1,16 @@
 language: go
 
 go:
-  - '1.11'
+  - '1.12.x'
 
 services:
   - docker
 
 env:
-  - BBLFSHD_VERSION=v2.11.8
+  - GO111MODULE=on BBLFSHD_VERSION=v2.12.1
```

Comment:
If the intent is to have multiple env vars setup by default (rather than adding rows to the build matrix), I find this intent to be more clear when  [`global:` key is used in `env`](https://docs.travis-ci.com/user/environment-variables/#global-variables)

```yml
env:
  global:
    - GO111MODULE=on
    - BBLFSHD_VERSION=v2.12.1
```

Reply:
Agree, this looks better :)